dotenv/env.go

```diff
@@ -8,19 +8,20 @@ import (
 	dotenv "github.com/mexisme/multiconfig/pathed"
 )
 
-// EnvAddConfig TODO
+//TODO EnvAddConfig
```

Comment:
If we are leaving TODOs in code, it's best we elaborate them in detail so future maintainers have some background on the issue.

Reply:
I agree with you, I just moved it to ahead making it more apparent, I didn't get anything from the repeated method name